drivers/ethernet/eth_gecko.c

```diff
@@ -0,0 +1,680 @@
+/*
+ * Copyright (c) 2019 Interay Solutions B.V.
+ * Copyright (c) 2019 Oane Kingma
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Silicon Labs EFM32 Giant Gecko 11 Ethernet driver.
+ * Limitations:
+ * - no statistics collection
+ * - no link monitoring through PHY interrupt
+ */
+
+#define LOG_MODULE_NAME eth_gecko
+#define LOG_LEVEL CONFIG_ETHERNET_LOG_LEVEL
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
```

Comment:
This is old way of setting up logging. Instead of this, you could just write
```
#include <logging/log.h>
LOG_MODULE_REGISTER(eth_gecko, CONFIG_ETHERNET_LOG_LEVEL);
```

Reply:
Changed